debian/control

```diff
@@ -392,6 +392,7 @@ Depends: ceph-base (= ${binary:Version}),
          ${misc:Depends},
          ${python3:Depends},
          ${shlibs:Depends},
+         libprotobuf23 <pkg.ceph.crimson>,
```

Comment:
libprotobuf23 seems to be the right package from jammy -- I don't see a way to do this more generally?